README.md

```diff
@@ -171,6 +175,60 @@ The `profile` object provides a set of attributes corresponding to user attribut
 
 Update your Callback URL in our Yoti Dashboard and you should be good to go!
 
+## AML Integration
+
+Yoti provides an AML (Anti Money Laundering) check service to allow a deeper KYC process to prevent fraud. This is a chargeable service, so please contact [sdksupport@yoti.com](mailto:sdksupport@yoti.com) for more information.
+
+Yoti will provide a boolean result on the following checks:
+
+* PEP list - Verify against Politically Exposed Persons list
+* Fraud list - Verify against  US Social Security Administration Fraud (SSN Fraud) list
+* Watch list - Verify against watch lists from the Office of Foreign Assets Control
+
+To use this functionality you must ensure your application is assigned to your Organisation in the Yoti Dashboard - please see here for further information.
+
+For the AML check you will need to provide the following:
+
+* Data provided by Yoti (please ensure you have selected the Given name(s) and Family name attributes from the Data tab in the Yoti Dashboard)
+  * Given name(s)
+  * Family name
+* Data that must be collected from the user:
+  * Country of residence (must be an ISO 3166 3-letter code)
+  * Social Security Number (US citizens only)
+  * Postcode/Zip code (US citizens only)
+
+### Consent
+
+Performing an Aml check on a person *requires* their consent.
+**You must ensure you have user consent *before* using this service.**
+
+### Code Example
+
+Given a YotiClient initialised with your SDK ID and KeyPair (see [Configuration](#configuration)) performing an AML check is a straightforward case of providing basic profile data.
+
+```javascript
+
+// Initiate user profile data.
+let country = new YotiEntity.Country('GBR');
```

Comment:
I'm a bit confused about the meaning of YotiEntity.
Also does Country need to be a type?

Reply:
any thoughts @laurent-yoti

Reply:
Feels like the term YotiEntity is here to act as an umbrella package domain name.
The code could just read something like:
```
import {Country} from YotiEntity
const country = new Country(...)
```
Yet not sure that Country is really related to Yoti, and then what is a YotiEntity?

Reply:
I am using `yoti` as a namespace as these are part of `yoti` package, and `entity` is a way to group those elements together, so I am following the folder name `yoti_entity`. I can understand `entity` is not widely used in `node` community, I can replace that folder name with `aml_type` or `aml_item`? Let me know your thought please.